static/src/js/components/ManageCollectionAssociation/ManageCollectionAssociation.jsx

```diff
@@ -105,13 +111,15 @@ const ManageCollectionAssociation = () => {
   })
 
   // Handles deleting selected collection
-  // if no collections selected, returns an error notification
   const handleDeleteAssociation = () => {
```

Comment:
May want to surface an error if can't delete -- const handleDeleteAssociation = () => {
```suggestion
  const handleDeleteAssociation = () => {
```
  setIsDeleting(true);
  deleteAssociationMutation({
    variables: {
      conceptId,
      associatedConceptIds: collectionConceptIds
    }
  }).catch((error) => {
    // Handle error
    console.error('Failed to delete association:', error);
    addNotification({
      message: 'Failed to delete association',
      variant: 'danger'
    });
  }).finally(() => {
    setIsDeleting(false);
  });
};